gpu/install_gpu_driver.sh

```diff
@@ -372,12 +372,11 @@ function main() {
   # This configuration should be ran on all nodes
   # regardless if they have attached GPUs
   configure_yarn
-
+  configure_yarn_nodemanager
+  configure_gpu_isolation
+    
   # Detect NVIDIA GPU
   if (lspci | grep -q NVIDIA); then
-    configure_yarn_nodemanager
```

Comment:
I think that without this change tests on 2.0 images will fail if GPUs are not attached to all worker nodes, because GPU resources will be configured for NodeManagers that do not have GPUs.

May you rebase your PR on master HEAD and I will kick off a presubmit.

Reply:
I added a change to only configure yarn gpu for gpu worker nodes and master. And rebased this PR on master HEAD